tests/test_dcos_e2e/scratch/test_ip_detect.py

```diff
@@ -0,0 +1,53 @@
+from pathlib import Path
+from textwrap import dedent
+
+from py.path import local  # pylint: disable=no-name-in-module, import-error
+
+from dcos_e2e.backends import ClusterBackend
+from dcos_e2e.cluster import Cluster
+from dcos_e2e.node import Role
+
+
+class TestCustomIPDetect:
```

Comment:
Instead, let's call this `TestCopyFiles` - the custom IP detect thing is just an implementation detail of the test.